test/functorch/test_vmap.py

```diff
@@ -4003,6 +4024,13 @@ def test():
                     continue
                 kwargs = sample_input.kwargs
                 is_batch_norm_and_training = is_batch_norm_training(op.name, kwargs)
+                out_dim = 0
+                if op.name.endswith("CustomOp"):
```

Comment:
String match is a little sketchy. Instead of ferrying the information over via a dictionary, could we do one of the following to reduce complexity?
- if the `op` is a CustomOp object, then we use the schema information to construct out_dims
- just special case numpy_split_copy_with_int here (it's the only one, right?)

Reply:
I changed to a special case for numpy_split_copy_with_int here.

We can't infer the `out_dim` from schema information alone, we have to look at the inputs as well, because in this case, `out_dim` depends on the inputs (the number of splits specifically).